Origami/src/components/user/nonghUserProfile.js

```diff
@@ -219,151 +241,155 @@ class NonGHUserProfileComponent extends React.Component {
-                                    ghost
-                                    onClick={() =>
-                                      this.toggleShowDataDialog({
-                                        type: "permalink",
-                                        content: `${
-                                          window.location.protocol
-                                        }//${window.location.host}${
-                                          this.state.permalinkHolder[
-                                            this.state.user.id
-                                          ][project.id].short_relative_url
-                                        }`
-                                      })
-                                    }
-                                  >
-                                    Permalink<Icon type="link" />
-                                  </Button>
-                                </Col>
-                                <Col span={10} offset={1}>
-                                  <Button
-                                    type="danger"
-                                    style={{ width: "100%" }}
-                                    ghost
-                                    onClick={() =>
-                                      this.toggleDeleteConfirmationDialog(
-                                        project.id
-                                      )
-                                    }
-                                  >
-                                    Delete <Icon type="delete" />
-                                  </Button>
-                                </Col>
-                              </Row>
-                              <br />
-                            </div>
-                          </Card>
-                        </Col>
-                      ))}
-                    </Row>
-                  </div>
-                ))}
-              </Row>
-            </div>
-          )}
-        </Content>
-        <Footer style={styles.footer}>
-          <strong>Origami</strong> - Created by{" "}
-          <a href="http://cloudcv.org/">Team CloudCV</a>
-        </Footer>
+        {this.state.demoLoading ? (
+          <div className="demoSpinner" style={demoSpinnerStyle}>
+            <BounceLoader color={'#33aadd'} size={80} />
+          </div>
+        ) : (
+          <Content>
+            {this.state.user && (
+              <div style={styles.contentDiv}>
+                <Row>
+                  {this.state.allDeployed.map(row => (
+                    <div key={Math.random()}>
+                      <Row>
+                        {row.map(project => (
+                          <Col span={5} offset={1} key={project.id}>
+                            <Card
+                              style={{ width: '100%' }}
+                              bodyStyle={{ padding: 0 }}
+                            >
+                              <div className="custom-card">
```

Comment:
There isn't a custom card component?

Reply:
Ill add the modified cards in some othe PR.This Pr is just for adding basic mydemo Page.I have raise a PR regarding modifying custom card component for Dashboard(#227 ).We will have similar card for my demo page too.